src/VisualStudio/Core/Def/Implementation/DesignerAttribute/VisualStudioDesignerAttributeService.cs

```diff
@@ -118,16 +118,15 @@ private async Task StartWorkerAsync()
 
             // Pass ourselves in as the callback target for the OOP service.  As it discovers
             // designer attributes it will call back into us to notify VS about it.
-            _connection = await client.CreateConnectionAsync(
-                WellKnownServiceHubService.RemoteDesignerAttributeService,
-                callbackTarget: this, cancellationToken).ConfigureAwait(false);
+            var connection = await client.CreateConnectionAsync<IRemoteDesignerAttributeService>(callbackTarget: this, cancellationToken).ConfigureAwait(false);
 
             // Now kick off scanning in the OOP process.
-            await _connection.RunRemoteAsync(
-                nameof(IRemoteDesignerAttributeService.StartScanningForDesignerAttributesAsync),
-                solution: null,
-                arguments: Array.Empty<object>(),
+            // If the call fails an error has already been reported and there is nothing more to do.
```

Comment:
reported by what mechanism?  do we mean reported to NFW? or reported to user? (or both?)

Reply:
all of the above. TryInvokeAsync takes care of it.